packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -2,35 +2,53 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
 
-/// Interface for localized resource values for the material widgets.
-///
-/// This class provides a default placeholder implementation that returns
-/// hard-coded American English values.
+import 'i18n/localizations.dart';
+
+/// Localized strings for the material widgets.
 class MaterialLocalizations {
```

Comment:
I think our default localization object should be really really simple. I'd prefer if we added the other languages as a separate package.

Reply:
Specifically, I'd like this class to be something you can extend without fear that you'll be dragging in code for any languages you don't know about.

Reply:
OK, I'll make this real interface, and add a class (DefaultMaterialLocalizations) that implements it.